packages/docs/src/stories/components/input.stories.ts

```diff
@@ -12,11 +12,11 @@ const { overrideArgs } = storybookHelpers('sd-input');
  * **Related templates**:
  * - [Input](?path=/docs/templates-input--docs)
  * - [Autocomplete](?path=/docs/templates-autocomplete--docs)
- * - [Tooltip](?path=/docs/templates-tooltip--docs)
+ * - [Input with Tooltip](?path=/docs/templates-tooltip--docs&args=open:!true#input%20with%20tooltip)
```

Comment:
```suggestion
 * - [Input with Tooltip](?path=/docs/templates-tooltip--docs#input%20with%20tooltip)
```

`args=open` can be removed, it's actually not doing anything as far as I see